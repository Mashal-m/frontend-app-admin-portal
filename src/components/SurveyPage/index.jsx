import React, { useEffect } from 'react';

const SurveyPage = () => {
  useEffect(() => {
    const widget = document.createElement('script');
    widget.src = 'https://widget.surveymonkey.com/collect/website/js/tRaiETqnLgj758hTBazgd_2BubSJEoVMSXyhpZ3VDb5_2BknEVoWfJFNQnAE6Sqt_2BFck.js';
    document.body.appendChild(widget);
  }, []);

  return <div id="smcx-sdk" />;
};

export default SurveyPage;
