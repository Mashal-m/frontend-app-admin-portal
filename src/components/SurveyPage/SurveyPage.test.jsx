import React from 'react';
import renderer from 'react-test-renderer';

import SurveyPage from './index';

describe('<SurveyPage />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create((
        <SurveyPage />
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
