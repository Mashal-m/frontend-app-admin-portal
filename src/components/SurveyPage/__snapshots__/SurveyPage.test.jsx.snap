// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SurveyPage /> renders correctly 1`] = `
<div
  id="smcx-sdk"
/>
`;
